pkgs/applications/science/electronics/csxcad/default.nix

```diff
@@ -0,0 +1,36 @@
+{ fetchFromGitHub, stdenv, cmake
+, fparser, tinyxml, hdf5, cgal_5, vtk
```

Comment:
fparser doesn't seem to exist:

```
anonymous function at /var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/mr-est/packet-spot-eval-1/pkgs/applications/science/electronics/csxcad/default.nix:1:1 called without required argument 'fparser'
```

Reply:
My mistake. fparser PR [here](https://github.com/NixOS/nixpkgs/pull/84654).

Reply:
I've rebased against master, so the fparser error should be fixed.